### PR TITLE
Added missing attributes to examples.

### PR DESCRIPTION
**Pull request by Strange-Satwik**
##### SUMMARY
Added the attributes to examples that are failing due to missing attributes.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME

Updated modules:
- cmci_action
- cmci_create
- cmci_delete
- cmci_get
- cmci_update

### CODE REVIEW

**Comment by andrewhughes101** [comment]
CI allowed

**Review by stewartfrancis — CHANGES_REQUESTED** [review]

File: docs/source/modules/cmci_action.rst [hunk 1]
```diff
@@ -374,16 +374,15 @@ Examples
    - name: install a bundle in a CICS region
      cmci_action:
        cmci_host: "example.com"
-       cmci_port: "12345"
```
[the remark this answers is not in the document]
Comment (Strange-Satwik) [reply docs/source/modules/cmci_action.rst:377]: removed the scheme

File: docs/source/modules/cmci_action.rst [hunk 2]
```diff
@@ -374,16 +374,14 @@ Examples
...
-       context: "plex_or_applid"
-       type: CICSBundle
-       action_name: install
-       resource:
+       cmci_port: 1234
+       context: "ABCDEFGH" # context is the name of your CICSplex in a CPSM environment or the applid of your region in an SMSS environment
+       scope: "CSYDEF/CSYSGRP" # scope only applies if you're in a CPSM environment and is either the name of a CICS system definition (CSYSDEF) or CICS system group (CSYSGRP)
```
Comment (stewartfrancis) [inline docs/source/modules/cmci_action.rst:379]: Again this needs to be an 8 char name.  Maybe use `IJKLMNOP` to follow the precedent you've already set?

**Review by stewartfrancis — CHANGES_REQUESTED** [review]
Sorry if I wasn't clear enough with those first comments.  I think you should apply the same pattern you've taken for `context` to `scope`.

File: docs/source/modules/cmci_action.rst (hunk 2 above)
Comment (Strange-Satwik) [reply docs/source/modules/cmci_action.rst:379]: Thinking of mentioning 8 characters long somewhere in the comment. What do you say

**Review by stewartfrancis — APPROVED** [review]

**Comment by stewartfrancis** [comment]
@andrewhughes101 this one's ready for a CI run now.  Do I have the ability to permit?  Looks like you tried previously!

**Comment by kiera-bennett** [comment]
CI allowed

**Comment by kiera-bennett** [comment]
CI allowed

**Pull request merged**